internal/clients/diffreviewer/circleci/circleci_service_test.go

```diff
@@ -385,6 +461,12 @@ func (c *circleCiTestSuite) TestWriteRepositoryComments() {
 				testCircleService.PrDetails.CommitSha,
 				gc,
 			)
+			mockLogger.EXPECT().Error(fmt.Sprintf(
+				"%s at %s on line %d",
+				gc.GetBody(),
+				gc.GetPath(),
+				tt.giveComments[index].LineNumber,
```

Comment:
unlike `github.PullRequestComment`, `github.RepositoryComment` does not contain a line number field, although it is still applicable for the user to trace the comment from circle workflow to their github repo with a line number e.g.:
![Screen Shot 2020-09-03 at 5 53 51 PM](https://user-images.githubusercontent.com/20913032/92177425-777f3680-ee0e-11ea-92f5-313de8ebb535.png)